include/robot_state_publisher/robot_state_publisher.h

```diff
@@ -99,6 +99,7 @@ class RobotStatePublisher
   std_msgs::msg::String model_xml_;
   bool description_published_;
   rclcpp::Publisher<std_msgs::msg::String>::SharedPtr description_pub_;
+  rclcpp::Clock::SharedPtr clock_;
```

Comment:
This breaks ABI by changing the size of `RobotStatePublisher`. This is fine for Eloquent, but is an issue for including in a dashing patch release. This bug might be important enough to warrant breaking ABI.

Reply:
It probably makes sense to break ABI and release this to Dashing, see also #21.